src/api/shared/create-redis.ts

```diff
@@ -17,7 +20,14 @@ export function createRedisFactory(redisUrl?: string): () => IORedis.Redis {
   let redis: IORedis.Redis | undefined = undefined;
   return () => {
     if (!redis) {
-      redis = new IORedis(redisUrl);
+      if (options.tls?.caPath && fs.existsSync(options.tls.caPath)) {
+        const { caPath, ...tls } = options.tls;
+        options.tls = {
+          ...tls,
+          ca: fs.readFileSync(options.tls.caPath, 'utf8'),
+        };
+      }
```

Comment:
similar to the comment above: I think having people pass in a base64-encoded version of the certificate instead of a path would be preferrable